### PR TITLE
Add manual testing for sort and edit DG, UG

### LINKED ISSUE #187: Update DG and UG and add manual testing for sort

### PR DESCRIPTION
**Pull request by miuna9890**
Fixes #187

### CODE REVIEW

**Review by haleemairfan — APPROVED** [review]
Looks good!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -301,10 +301,17 @@ Priorities: High (Must-Have), Medium (Nice-to-Have), Low (Could-Have), Trivial (
...
+1. User enters sort contact command with the field name.
 2. Cher shows list of contacts sorted alphabetically by name.
 3. Cher shows success message that list has been sorted by name.<br>
-Use case ends
+   Use case ends.
+
+**Extensions**
+* 1a. User leaves the sort field empty.
+    - 1a1. Cher detects error in user input and shows correct input format.
+* 2a. User enters invalid field.
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:312]: i think this is 1b. ? since it will happen after step 1

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -301,10 +301,17 @@ Priorities: High (Must-Have), Medium (Nice-to-Have), Low (Could-Have), Trivial (
...
+    - 2a1. Cher detects error in user input and shows correct input format. <br>
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:313]: this 1c. ?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -459,6 +466,54 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Sorting persons
+
+1. Prerequisites: <br>
+Execute the following:
+   - add n/Charlie s/m r/student p/567893 a/Bukit batok e/nus@dfsh.dsfvc  t/tag1 
+   - add n/anna s/f r/parent p/23456 a/Jurong e/sutd@dfsh.dsfvc  t/tag1
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:474]: phone numbers need to be 8 digits long

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -459,6 +466,54 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `sort name` <br>
+      Expected: Feedback box will show message: `Sorted by name` <br>
+      Contact entries will show list sorted in case-insensitive order based on ASCII value of each character in 
+      the name, in the following order: anna, Benet, Charlie
+3. Sorting list by role
+   1. Prerequisites: <br>
+      List all persons using the `list` command.
+   2. Test case: `sort role` <br>
+      Expected: Feedback box will show message: `Sorted by role` <br>
+      Contact entries will show list sorted in case-insensitive order based on ASCII value of each character in
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:487]: Maybe can mention that all parents will be sorted before students?

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -459,6 +466,54 @@ testers are expected to do more *exploratory* testing.
...
+    1. Prerequisites: <br>
+       List all persons using the `list` command.
+    2. Test case: `sort email` <br>
+       Expected: Feedback box will show message: `Sorted by email` <br>
+       Contact entries will show list sorted in case-insensitive order based on ASCII value of each character in
+       the email, in the following order: Charlie, Benet, anna 
+6. Sorting list by address
+    1. Prerequisites: <br>
+       List all persons using the `list` command.
+    2. Test case: `sort name` <br>
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:506]: Test case shld be sort address

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -459,6 +466,54 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: Feedback box will show message: `Sorted by address` <br>
+       Contact entries will show list sorted in case-insensitive order based on ASCII value of each character in
+       the address, in the following order: Charlie, anna, Benet
+7. Sorting list by invalid field
+    1. Prerequisites: <br>
+       List all persons using the `list` command.
+    2. Test case: `sort k` <br>
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:513]: Maybe mention that contact entries will not be rearranged/remain unchanged

File: docs/UserGuide.md [hunk 7]
```diff
@@ -318,13 +322,14 @@ Furthermore, certain edits can cause the Cher to behave in unexpected ways (e.g.
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME r/ROLE p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho r/student p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Secondary 1`
+**Add** | `add n/NAME s/SEX r/ROLE p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho r/student p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Secondary 1`
```
Comment (yyueyii) [inline docs/UserGuide.md:325]: The example is missing the sex field

**Review by yyueyii — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -301,10 +301,17 @@ Priorities: High (Must-Have), Medium (Nice-to-Have), Low (Could-Have), Trivial (
...
-1. User enters sort contact command with the predicate name.
+1. User enters sort contact command with the field name.
 2. Cher shows list of contacts sorted alphabetically by name.
 3. Cher shows success message that list has been sorted by name.<br>
-Use case ends
+   Use case ends.
+
+**Extensions**
+* 1a. User leaves the sort field empty.
+    - 1a1. Cher detects error in user input and shows correct input format.
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:311]: Missing Use case ends

**Review by yyueyii — APPROVED** [review]
LGTM!

**Pull request merged**